internal/model/model.go

```diff
@@ -1222,6 +1218,7 @@ nextSub:
 		TempNamer:     defTempNamer,
 		TempLifetime:  time.Duration(m.cfg.Options().KeepTemporariesH) * time.Hour,
 		CurrentFiler:  cFiler{m, folder},
+		MtimeRepo:     db.NewVirtualMtimeRepo(m.db, folderCfg.ID),
```

Comment:
I don't like the word repo, perhaps just mtimes/virtualmtimes? Given that's the best thing I got to pick on, I guess the rest of the code is good.